_rules/letter-spacing-not-important-24afc2.md

```diff
@@ -0,0 +1,161 @@
+---
+id: 24afc2
+name: Letter spacing in `style` attributes is not `!important`
+rule_type: atomic
+description: |
+	This rule checks that the `style` attribute is not used to prevent adjusting `letter- spacing` by using `!important`, except if it's at least `0.12` times the font size.
```

Comment:
```suggestion
	This rule checks that the `style` attribute is not used to prevent adjusting `letter-spacing` by using `!important`, except if it's at least `0.12` times the font size.
```